zio-cli/shared/src/main/scala/zio/cli/Command.scala

```diff
@@ -69,6 +69,11 @@ object Command {
     }
   }
 
+  private def splitForcedArgs(args: List[String]): (List[String], List[String]) = {
+    val (forcedArgs, remainingArgs) = args.span(_ != "--")
```

Comment:
Should it be `optionsAndArgs, forcedArgs`?

Reply:
yes